src/components/Semester.vue

```diff
@@ -267,9 +261,9 @@ export default {
         let inPreviousQuarter = false;
         if (subj2 !== undefined) {
           inPreviousQuarter = s.semester === this.index &&
-                                  subjInQuarter2 &&
-                                  subj2.quarter_information !== undefined &&
-                                  subj2.quarter_information.split(',')[0] === '0';
+            subjInQuarter2 &&
+            subj2.quarter_information !== undefined &&
+            subj2.quarter_information.split(',')[0] === '0';
```

Comment:
I am personally a fan of putting the `&&`s at the beginning of the line and think that's a lot more readable, but if the linter says this is better then maybe it is

Reply:
Personal preference, but I like keeping the `&&` and other operators on the same line.